File: app/src/main/java/com/example/ontime/MyOrdersActivity.java
package com.example.ontime;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

public class MyOrdersActivity extends AppCompatActivity {
    private RecyclerView mRecyclerView;
    private MyOrderAdapter mAdapter;
    private RecyclerView.LayoutManager mLayoutManager;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_my_orders);
        final ArrayList<MyOrderItem> myOrderList = new ArrayList<>();
        myOrderList.add(new MyOrderItem(R.drawable.bg5,"Aroma","Пицца Пепперони", "Уже в пути","3х"));
        myOrderList.add(new MyOrderItem(R.drawable.bg5,"CoffeeBOOM","Гнездо глухаря", "Обрабатывается","4х"));

        mRecyclerView = findViewById(R.id.my_orders_recycler);
        mAdapter = new MyOrderAdapter(myOrderList);

        mRecyclerView.setHasFixedSize(true);
        mLayoutManager = new LinearLayoutManager(this);
        mRecyclerView.setLayoutManager(mLayoutManager);
        mRecyclerView.setAdapter(mAdapter);



    }
}

File: app/src/main/java/com/example/ontime/MyOrdersActivity.kt
package com.example.ontime

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import kotlinx.android.synthetic.main.activity_my_orders.*
import java.util.*

class MyOrdersActivity : AppCompatActivity() {
    private var mAdapter: MyOrderAdapter? = null
    private var mLayoutManager: RecyclerView.LayoutManager? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_my_orders)
        val myOrderList = ArrayList<MyOrderItem>()
        myOrderList.add(MyOrderItem(R.drawable.bg5, "Aroma", "Пицца Пепперони", "Уже в пути", "3х"))
        myOrderList.add(MyOrderItem(R.drawable.bg5, "CoffeeBOOM", "Гнездо глухаря", "Обрабатывается", "4х"))
        mAdapter = MyOrderAdapter(myOrderList)
        my_orders_recycler.setHasFixedSize(true)
        mLayoutManager = LinearLayoutManager(this)
        my_orders_recycler.setLayoutManager(mLayoutManager)
        my_orders_recycler.setAdapter(mAdapter)
    }
}